### PR TITLE
Bugfix/ add schema Id to create resource request

### PR DESCRIPTION
**Pull request by kenjinp**
fixes: https://github.com/BlueBrain/nexus/issues/901

### CODE REVIEW

File: src/shared/containers/ResourceFormContainer.tsx [hunk 1]
```diff
@@ -11,10 +11,8 @@ const ResourceFormContainer: React.FunctionComponent<{
 }> = ({ orgLabel, projectLabel }) => {
   const nexus = useNexusContext();
   const createResource = (schemaId: string, payload: ResourcePayload) => {
-    // CANNOT create resource with schemaID
-    // TODO: update after fix
-    // https://github.com/BlueBrain/nexus/issues/788
-    return nexus.Resource.create(orgLabel, projectLabel, payload);
+    // TODO: bump @bbp/nexus-sdk
```
Comment (julienmachon) [inline src/shared/containers/ResourceFormContainer.tsx]: :thinking: 
sdk 1.1.8 was release with `schemaId` support https://github.com/BlueBrain/nexus-js/releases/tag/v1.1.8

**Comment by julienmachon** [comment]
The creation of an aggregate view doesn't work for me:
Screenshot from 2019-11-26 08-33-30

File: src/shared/containers/ResourceFormContainer.tsx (hunk 1 above)
Comment (kenjinp) [reply src/shared/containers/ResourceFormContainer.tsx]: you're right 🙊

**Comment by julienmachon** [comment]
> The creation of an aggregate view doesn't work for me:
> Screenshot from 2019-11-26 08-33-30

This is fine actually! Works now

**Pull request merged**